packages/react-components/react-nav-preview/src/components/NavGroup/useNavGroup.styles.ts

```diff
@@ -0,0 +1,245 @@
+    `,
+      zIndex: 1,
+    },
+    { enableOutline: true },
+  ),
+});
+
+/** Indicator styles for when pending selection */
+const usePendingIndicatorStyles = makeStyles({
+  base: {
+    ':hover::before': {
+      backgroundColor: tokens.colorNeutralStroke1Hover,
+      ...shorthands.borderRadius(tokens.borderRadiusCircular),
+      content: '""',
+      position: 'absolute',
+    },
+    ':active::before': {
+      backgroundColor: tokens.colorNeutralStroke1Pressed,
+      ...shorthands.borderRadius(tokens.borderRadiusCircular),
+      content: '""',
+      position: 'absolute',
+    },
+    '@media (forced-colors: active)': {
+      ':hover::before': {
+        backgroundColor: 'Highlight',
+      },
+      ':active::before': {
+        backgroundColor: 'Highlight',
+      },
+    },
+  },
+});
+
+const useActiveIndicatorStyles = makeStyles({
+  base: {
+    '::after': {
+      backgroundColor: tokens.colorTransparentStroke,
+      ...shorthands.borderRadius(tokens.borderRadiusCircular),
+      content: '""',
+      position: 'absolute',
+      zIndex: 1,
+    },
+  },
+  selected: {
+    '::after': {
+      backgroundColor: tokens.colorCompoundBrandStroke,
+    },
+    ':hover::after': {
+      backgroundColor: tokens.colorCompoundBrandStrokeHover,
+    },
+    ':active::after': {
+      backgroundColor: tokens.colorCompoundBrandStrokePressed,
+    },
+    '@media (forced-colors: active)': {
+      '::after': {
+        backgroundColor: 'ButtonText',
+      },
+      ':hover::after': {
+        backgroundColor: 'ButtonText',
+      },
+      ':active::after': {
+        backgroundColor: 'ButtonText',
+      },
+    },
+  },
+});
+
+/**
+ * Styles for the icon slot.
+ */
+const useIconStyles = makeStyles({
+  base: {
+    gridColumnStart: 1,
+    gridRowStart: 1,
+    alignItems: 'center',
+    display: 'inline-flex',
+    justifyContent: 'center',
+    ...shorthands.overflow('hidden'),
+    [`& .${iconClassNames.filled}`]: {
+      display: 'none',
+    },
+    [`& .${iconClassNames.regular}`]: {
+      display: 'inline',
+    },
+  },
+  selected: {
+    [`& .${iconClassNames.filled}`]: {
+      display: 'inline',
+    },
+    [`& .${iconClassNames.regular}`]: {
+      display: 'none',
+    },
+  },
+});
+
+/**
+ * Styles for the content slot (children)
+ */
+const useContentStyles = makeStyles({
+  base: {
+    ...typographyStyles.body1,
+    ...shorthands.overflow('hidden'),
+    // content padding is the same for medium & small, horizontal & vertical
+    ...shorthands.padding(tokens.spacingVerticalNone, tokens.spacingHorizontalXXS),
+  },
+  selected: {
+    ...typographyStyles.body1Strong,
+  },
+  noIconBefore: {
+    gridColumnStart: 1,
+    gridRowStart: 1,
+  },
+  iconBefore: {
+    gridColumnStart: 2,
+    gridRowStart: 1,
+  },
+  placeholder: {
+    visibility: 'hidden',
+  },
+});
+
+/**
+ * Apply styling to the Tab slots based on the state
+ */
+export const useNavGroupStyles = (state: NavGroupState): NavGroupState => {
```

Comment:
you should add unstable suffix
```suggestion
export const useNavGroupStyles_unstable = (state: NavGroupState): NavGroupState => {
```